molecule/default/yamllint.yml

```diff
@@ -9,3 +9,6 @@ rules:
     level: error
   truthy: disable
   document-start: disable
+  line-length:
+    level: warning
+    allow-non-breakable-inline-mappings: true
```

Comment:
is this related to ubuntu suse support?
Is this related to molecule v3 support?